build.gradle

```diff
@@ -0,0 +1,63 @@
+group 'org.bitbucket.lykke'
+version '1.0.0-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.2.10'
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+apply plugin: 'java'
+apply plugin: 'kotlin'
+apply plugin: 'maven'
+
+repositories {
+    mavenCentral()
+    maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+}
+
```

Comment:
suggest to add         freeCompilerArgs = ["-Xno-param-assertions", "-Xno-call-assertions"]
and         jvmTarget = "1.8"

Reply:
fixed